content/python/concepts/functions/functions.md

```diff
@@ -82,27 +85,41 @@ Function parameters can also be initialized to a default value.
 In the function `calc_total()`, the default value of `discount` is 10.
 
 - When the `discount` value is explicitly specified in the function call, that value is used.
-- Else, the default value of 10 is used.
+- Otherwise, the default value of 10 is used.
 
 ```py
 def calc_total(amount, discount=10):
   total = amount * (1 - 0.01*discount)
   return total
 
-calc_total(100)
-
-calc_total(250, 5)
+calc_total(100)     # Output: 90.0
+calc_total(250, 5)  # Output: 237.5
 ```
 
-## Functions with Variable Number of Parameters
+## Keyword Arguments
+
+Unless specified otherwise, the arguments passed into a function are assigned to each parameter in order in which they appear in the function call. Thus, they are also known as "positional arguments".
```

Comment:
```suggestion
Unless specified otherwise, the arguments passed into a function are assigned to each parameter in the order in which they appear in the function signature. Thus, they are also known as "positional arguments".
```